### ISSUE TITLE
Create hepdata spider

### ISSUE THREAD

**Issue by david-caro** [inspirehep/hepcrawl#90]

**Comment by david-caro** [comment]
@kaplun Hey, this should be done before we shut down legacy too right?

**Comment by kaplun** [comment]
Yes, not too difficult luckily. Current code on legacy is here:
https://github.com/inspirehep/inspire/blob/master/bibtasklets/bst_hepdata.py

**Comment by michamos** [comment]
This requires a proper schema for data though. We should discuss that with @suenjedt at some point.

**Comment by suenjedt** [comment]
Schema is easy I think. cc'ing @ArtemisLav who is the expert

**Comment by kaplun** [comment]
Yeah actually Schema could very well be a subset of Literature schema, plus some dedicated field for the datatype.
